shared/common-adapters/toast.desktop.tsx

```diff
@@ -13,6 +13,7 @@ const Toast = (props: Props) => (
     propagateOutsideClicks={true}
     position={props.position}
     containerStyle={styles.float}
+    disableEscapeKey={true}
```

Comment:
its possible for a toast to show and then you get it as the topmost item in the escape handling stack. That stack isn't aware of visibility / etc so it'd just lose the escape. Instead of changing that system lets just not allow toasts to eat keys